rust_src/src/base64.rs

```diff
@@ -55,8 +55,8 @@ pub extern "C" fn base64_encode_1(
 }
 
 /// Base64-decode the data at FROM of LENGTH bytes into TO.  If MULTIBYTE, the
-/// decoded result should be in multibyte form.  If NCHARS_RETURN is not NULL,
-/// store the number of produced characters in *NCHARS_RETURN.
+/// decoded result should be in multibyte form.  If `NCHARS_RETURN` is not NULL,
```

Comment:
This is inconsistent; either all uppercase VARS should be markdownified, or none.

Since this style is prevalent everywhere in Emacs (and is mandatory in docstrings), I wouldn't bother changing these.